draft-davidson-pp-protocol.md

```diff
@@ -163,6 +168,46 @@ We assume that all protocol messages are encoded into raw byte format
 before being sent. We use the TLS presentation language {{RFC8446}} to
 describe the structure of protocol data types and messages.
 
+# Background
+
+We discuss the core motivation behind the protocol along with the
+guarantees and assumptions that we make in this document.
+
+## Motivating use-cases
+
+The Privacy Pass protocol was originally developed to provide anonymous
+authorization of users of anonymity-preserving tools on the Internet,
+such as Tor. In particular, the protocol allows clients to reveal
+authorization tokens that they have been issued without linking the
+authorization to the actual issuance event. This means that the tokens
+cannot be used to link the browsing patterns of clients that reveal
+tokens.
+
+The Privacy Pass protocol has a number of use cases in practice. See
+{{DGSTV18}}, {{TrustTokenAPI}}, {{PrivateStorage}}, {{OpenPrivacy}}, and
+{{Brave}} for examples.
+
+## Anonymity and security guarantees
+
+As mentioned above, the protocol provides anonymity-preserving tokens
+for authorizing clients. Throughout this document, we use the terms
+"anonymous", "anonymous-preserving" and "anonymity" to refer to the core
+security guarantee of the protocol. This guarantee is the following
+statement.
+
+- A token issued by a server key and subsequently redeemed is
+  indistinguishable from any other token issued under the same key.
+
+We also require that clients cannot forge tokens, as otherwise Privacy
+Pass would have little value as an authorization protocol. This
+requirement is stated as follows.
+
+- Any client that is issued `N` tokens under a given server key cannot
+  redeem more than `N` valid tokens.
```

Comment:
```suggestion
Privacy Pass also prohibits clients from forging tokens, as otherwise the protocol
would have little value as an authorization protocol. Informally, this means any client 
that is issued `N` tokens under a given server key cannot redeem more than `N` valid 
tokens.
```